src/actions/walletActions.js

```diff
@@ -155,16 +156,22 @@ export function autoSelectCoins() {
   };
 }
 
-export function spendSlices(inputs, changeSlice) {
+/**
+ * @description Given the state of the transaction store, check status of
+ * the slices being used in the spend tx and update them once changed state is confirmed.
+ * @param {Object} changeSlice - the slice that was chosen as the change output
+ */
+export function updateTxSlices(changeSlice, retries = 5) {
   return async (dispatch, getState) => {
     const {
       settings: { network },
       client,
       spend: {
-        transaction: { changeAddress },
+        transaction: { changeAddress, inputs, txid },
       },
       wallet: {
-        deposits: { nextNode },
+        deposits: { nextNode, nodes: depositNodes },
+        change: { nodes: changeNodes },
```

Comment:
Need these to compare if the network call reflects a change compared to the current state of the same slice.